src/data/pages/contribute/index.js

```diff
@@ -171,6 +172,90 @@ const data = {
       ],
     },
   },
+  'pt-br': {
+    hero: {
+      title: 'Como Contribuir com o eBPF',
+      description:
+        'O eBPF consiste em muitas comunidades, incluindo o _runtime_ do eBPF no kernel Linux, vários SDKs e bibliotecas de desenvolvimento, uma série de grandes projetos usando eBPF e um amplo conjunto de projetos e ferramentas menores. Existem, portanto, muitas maneiras de contribuir com a comunidade eBPF e de se envolver. A lista de ideias a seguir pode servir como ponto de partida sobre como você pode se envolver e contribuir com a comunidade.',
```

Comment:
```suggestion
        'O eBPF consiste em muitas comunidades, incluindo o runtime do eBPF no kernel Linux, vários SDKs e bibliotecas de desenvolvimento, uma série de grandes projetos usando eBPF e um amplo conjunto de projetos e ferramentas menores. Existem, portanto, muitas maneiras de contribuir com a comunidade eBPF e de se envolver. A lista de ideias a seguir pode servir como ponto de partida sobre como você pode se envolver e contribuir com a comunidade.',
```

Reply:
I used Markdown formatting when I made this suggestion but it doesn't match the file format.